modules/vocabularies/src/main/frontend/src/vocabularyDetails.jsx

```diff
@@ -60,16 +63,32 @@ const useStyles = makeStyles(theme => ({
   dialogTitle: {
     marginRight: theme.spacing(5)
   },
+  browseAction: {
+    margin: theme.spacing(1),
+    textTransform: "none",
+    marginRight: theme.spacing(37)
```

Comment:
I don't think we need a magic number, let's see if this works:
```suggestion
    marginRight: "auto"
```

Reply:
Done in 33f5307 .